CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ Since v0.0.10, this Changelog is formatted according to the [Common Changelog][c
 
 ## UNRELEASED
 
+### Changed 
+- ([#774](https://github.com/demos-europe/demosplan-ui/pull/774)) Add properties such as nodeIsSelected, nodeType, and nodeId to each node object and recursively updates their children nodes. This eliminates the need for intermediate steps, resulting in enhanced overall performance ([@sakutademos](https://github.com/sakutademos))
```

Comment:
I suggest to not be too explicit in terms of implementation details here. "Improve render performance of ..." would be ok, too.

Reply:
right, done here https://github.com/demos-europe/demosplan-ui/pull/774/commits/d8a566ccdc7d2f3dfdd1ad24a2cc7f31afc3f77f